### DISCUSSION TITLE
Formatting timestamptz miss zeros by default

### DISCUSSION THREAD

**Discussion by KES777** [dbeaver/dbeaver#12827] [Q&A]
#### System information: 
- Operating system (distribution) and version Linux mint 17.3
- DBeaver version 21.0.3

#### Describe the problem you're observing:
![image](https://user-images.githubusercontent.com/799021/116438509-30d7c400-a857-11eb-879a-6f29f5a1ea16.png)

data is not adjusted

it hard to check values visually, because data become not adjusted

**Comment by kseniiaguzeeva** [comment]
The last value has bigger length and cells are right aligned. So it is not a bug. Do you have any alternative how to show such values?

**Comment by KES777** [comment]
Yes, alternative is to show all digits:
`2019-02-27 17:47:16.500310+02`

**Comment by kseniiaguzeeva** [comment]
Please tell me your database and column type. Which date format do you use (you can see it in Preferences>Data Editor>Data Formats)?

**Comment by KES777** [comment]
PostgreSQL, timestamptz

![image](https://user-images.githubusercontent.com/799021/119791922-f0af4400-bedd-11eb-889d-3d810079ac46.png)

**Comment by KES777** [comment]
I have changed format:
![image](https://user-images.githubusercontent.com/799021/119792288-408e0b00-bede-11eb-9583-ef658ca0a824.png)

1. It would be better if more robust defaults will be by default ;-)
2. Also `Sample` does not show that this format displays trailing zeros

**Comment by KES777** [comment]
hm. probably I understand the reason, this come from locale settings.

But be notified, still most defaults we can get from locale settings, but this is special software, so in this case locale settings are not desired. PC users in real life ignore fraction of seconds, but DB developers - not.

Also third question: 
3. we saw that by default there no `ffffff` at pattern settings, Why second fraction were displayed?

**Comment by Matvey16** [comment]
Answering your third question. You have checked an option to disable date/time formatting, so the formatting pattern is not applied.

**Comment by KES777** [comment]
@Matvey16 Ok, so (about third question) why enabling that option change nothing?

**Comment by Matvey16** [comment]
I tried to explain how it works here #12618 
Let me know if it is still confusing
